src/aggregation/metric/cardinality.rs

```diff
@@ -191,6 +191,12 @@ impl SegmentCardinalityCollector {
                         let val = f64_to_u64(*val);
                         self.cardinality.sketch.insert_any(&val);
                     }
+                    Key::U64(val) => {
```

Comment:
sorry for using this PR.. 

`.expect("Found placeholder term_ord but `missing` is None");`

I think we are just supposed to ignore the value.